src/doc/src/reference/source-replacement.md

```diff
@@ -145,7 +153,9 @@ modifications.
 ## Git sources
 
 Git sources represent repositories used by [git-based dependencies]. They're
-used by `cargo vendor` to replace git-based dependencies with local directories.
+used to replace git-based dependencies with alternative sources during vendoring.
```

Comment:
Isn't this for specifying the git dependency to then replace it with another source, like `directory`?